### PR TITLE
001 start bundle structure

### PR DESCRIPTION
**Pull request by koliagindm**

### CODE REVIEW

File: src/MyShop/CommonBundle/Controller/AbstractController.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace MyShop\CommonBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class AbstractController extends Controller
```
Comment (koliagindm) [inline src/MyShop/CommonBundle/Controller/AbstractController.php]: Done.

File: src/MyShop/CommonBundle/Resources/config/routing.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+common_homepage:
```
Comment (koliagindm) [inline src/MyShop/CommonBundle/Resources/config/routing.yml]: Done.

File: src/MyShop/CommonBundle/Resources/views/Default/index.html.twig [hunk 3]
```diff
@@ -0,0 +1 @@
+Hello {{ name }}!
```
Comment (koliagindm) [inline src/MyShop/CommonBundle/Resources/views/Default/index.html.twig]: Done.

File: src/MyShop/WebsiteBundle/Controller/IndexController.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?php
+   
+namespace MyShop\WebsiteBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Response;
+
+class IndexController extends Controller
```
Comment (koliagindm) [inline src/MyShop/WebsiteBundle/Controller/IndexController.php]: Done.

File: src/MyShop/CommonBundle/Resources/config/services.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+parameters:
+    myshop.common.controller.class: MyShop\CommonBundle\Controller\AbstractController
+services:
+    myshop.common.controller:
+        class: %myshop.common.controller.class%
```
Comment (koliagindm) [inline src/MyShop/CommonBundle/Resources/config/services.yml]: Done.

File: src/MyShop/WebsiteBundle/Controller/IndexController.php [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+use Symfony\Component\HttpFoundation\Response;
+
+class IndexController extends Controller 
+{
+    /**
+     * @Route("/")
+     */
+    public function indexAction() 
+    {
+        return new Response("<html><head></head><body>Hello, world</body></html>");
```
Comment (koliagindm) [inline src/MyShop/WebsiteBundle/Controller/IndexController.php]: Done.

File: src/MyShop/WebsiteBundle/Resources/views/Default/index.html.twig [hunk 7]
```diff
@@ -0,0 +1 @@
+Hello {{ name }}!
```
Comment (koliagindm) [inline src/MyShop/WebsiteBundle/Resources/views/Default/index.html.twig]: Done.

File: src/MyShop/CommonBundle/Resources/config/services.yml (hunk 5 above)
Comment (koliagindm) [inline src/MyShop/CommonBundle/Resources/config/services.yml]: Done.

File: app/config/routing.yml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+website:
+    resource: "@WebsiteBundle/Controller/"
```
Comment (koliagindm) [inline app/config/routing.yml]: Done.

File: src/MyShop/CommonBundle/DependencyInjection/CommonExtension.php [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    /**
+     * @param array $configs
+     * @param ContainerBuilder $container
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('services.yml');
+    }
+}
```
Comment (EduardTrayan) [inline src/MyShop/CommonBundle/DependencyInjection/CommonExtension.php]: Please add new line

File: src/MyShop/WebsiteBundle/DependencyInjection/WebsiteExtension.php [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+    /**
+     * @param array $configs
+     * @param ContainerBuilder $container
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('services.yml');
+    }
+}
```
Comment (EduardTrayan) [inline src/MyShop/WebsiteBundle/DependencyInjection/WebsiteExtension.php]: Please add new line

File: src/MyShop/WebsiteBundle/Resources/config/routing.yml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+website:
+    resource: "@WebsiteBundle/Controller/"
+    type: annotation
```
Comment (EduardTrayan) [inline src/MyShop/WebsiteBundle/Resources/config/routing.yml]: Please add new line

File: src/MyShop/CommonBundle/DependencyInjection/CommonExtension.php (hunk 9 above)
Comment (koliagindm) [inline src/MyShop/CommonBundle/DependencyInjection/CommonExtension.php]: Done.

File: src/MyShop/WebsiteBundle/Resources/config/routing.yml (hunk 11 above)
Comment (koliagindm) [inline src/MyShop/WebsiteBundle/Resources/config/routing.yml]: Done.

**Pull request merged**